includes/wikia/services/UserStatsService.class.php

```diff
@@ -103,41 +95,25 @@ public function increaseEditsCount() {
 	 * - number of edits in current week
 	 * @return array
 	 */
-	public function getStats( $flags = 0 ) {
+	public function getStats() {
 		$stats = WikiaDataAccess::cache(
-			self::getUserStatsMemcKey( $this->userId, $this->getWikiId() ),
+			self::getUserStatsMemcKey( $this->userId, $this->wikiId ),
 			self::CACHE_TTL,
-			function () use ( $flags ) {
+			function () {
 				$stats = $this->loadUserStatsFromDB();
 
 				if ( !isset( $stats['editcount'] ) ) {
-					// editcount not set yet, calculate it
-					$stats['editcount'] = $this->calculateEditCountWiki( $flags );
-				}
-				else {
-					// make sure this value is an integer
-					$stats['editcount'] = (int) $stats['editcount'];
-				}
-
-				if ( $stats['editcount'] === 0 ) {
+					// editcount not set yet, so user has no edits
```

Comment:
Are we sure that's always true? Did we initialise values in the table with proper user edit counters on each wiki?

Reply:
SQL logs indicate this gets called for users with 0 edits on the wiki